cmd/tempo-cli/cmd-rewrite-blocks.go

```diff
@@ -43,48 +44,55 @@ func (cmd *dropTraceCmd) Run(ctx *globalOptions) error {
 		return err
 	}
 
-	id, err := util.HexStringToTraceID(cmd.TraceID)
-	if err != nil {
-		return err
+	traceIDs := strings.Split(cmd.TraceID, ",")
+	ids := make([]common.ID, 0, len(traceIDs))
+	for _, traceID := range traceIDs {
+		id, err := util.HexStringToTraceID(traceID)
+		if err != nil {
+			return err
+		}
+		ids = append(ids, id)
 	}
 
-	blocks, err := blocksWithTraceID(context.Background(), r, cmd.TenantID, id)
-	if err != nil {
-		return err
-	}
+	for _, id := range ids {
+		blocks, err := blocksWithTraceID(context.Background(), r, cmd.TenantID, id)
```

Comment:
I am thinking the following approach:

build a list of unique blocks that contains these IDs, one trace ID can be in multiple blocks and one block can have multiple trace IDs that we want to delete.

once we have a unique list of blocks that contain these traceIDs, we can go over each block, and then delete all the trace IDs and push that block. this means that we are only rewriting a block only once for all the traceIDs that needs to be deleted from it.